### ISSUE TITLE
merging path removes marker-end

### ISSUE THREAD

**Issue by JoKalliauer** [svg/svgo#872]
Processing
```svg
<g marker-end="url(#a)"><path d="M11 20v8"/><path d="M15 30v-8"/></g>
```
as in:
[https://commons.wikimedia.org/wiki/File:Hippocampus_connections_libRSVGBug.svg](https://commons.wikimedia.org/wiki/File:Hippocampus_connections_libRSVGBug.svg)
with` --enable=mergePaths` (default)
leads to:
```svg
<g fill="none" marker-end="url(#a)" stroke="#000" transform="scale(10)"><path d="M11 20v8M15 30v-8M10 40v13"/></g>
```
which removes the endmarker.

This is simmilar to  #958

**Comment by TimothyGu** [comment]
I am experience this as well.

**Comment by TrySound** [comment]
Fixed in v2.2.1
